hack/quickstart-setup.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+#
+# Copyright 2021 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+if [ -z $KUADRANT_ORG ]; then
+  KUADRANT_ORG=${KUADRANT_ORG:="kuadrant"}
+fi
+if [ -z $KUADRANT_BRANCH ]; then
+  KUADRANT_BRANCH=${KUADRANT_BRANCH:="main"}
+fi
+if [ -z $MGC_BRANCH ]; then
+  MGC_BRANCH=${MGC_BRANCH:="main"}
+fi
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+source /dev/stdin <<< "$(curl -s https://raw.githubusercontent.com/adam-cattermole/multicluster-gateway-controller/update-quickstart/hack/.quickstartEnv)"
+source /dev/stdin <<< "$(curl -s https://raw.githubusercontent.com/adam-cattermole/multicluster-gateway-controller/update-quickstart/hack/.deployUtils)"
+
+KUADRANT_IMAGE="quay.io/${KUADRANT_ORG}/kuadrant-operator:latest"
+KUADRANT_REPO="github.com/${KUADRANT_ORG}/kuadrant-operator.git"
+KUADRANT_REPO_RAW="https://raw.githubusercontent.com/${KUADRANT_ORG}/kuadrant-operator/${KUADRANT_BRANCH}"
+KUADRANT_DEPLOY_KUSTOMIZATION="${KUADRANT_REPO}/config/deploy"
+KUADRANT_GATEWAY_API_KUSTOMIZATION="${KUADRANT_REPO}/config/dependencies/gateway-api"
+KUADRANT_ISTIO_KUSTOMIZATION="${KUADRANT_REPO}/config/dependencies/istio/sail"
+KUADRANT_CERT_MANAGER_KUSTOMIZATION="${KUADRANT_REPO}/config/dependencies/cert-manager"
+KUADRANT_METALLB_KUSTOMIZATION="${KUADRANT_REPO}/config/metallb"
+
+set -e pipefail
+
+if [[ "${KUADRANT_BRANCH}" != "main" ]]; then
```

Comment:
I wonder if this would be better named ad `KUADRANT_RELEASE` that way the doc could say set ```export KUADRANT_RELEASE=v0.6.0```